src/canvas/FreeLinePath.tsx

```diff
@@ -0,0 +1,45 @@
+import { PatternPath } from "./PatternPath";
+import { PatternPathType, ToolType } from "./Enums";
+import { Curve } from './Curve';
+import { CurveFitter } from './CurveFitter';
+
+export class FreeLinePath extends PatternPath {
+    private _fittedCurve: Curve | null;
+    
+    constructor (pathType: PatternPathType) {
+        super(pathType, ToolType.Freeline);
+        
+        this._fittedCurve = null;
+    }
+
+    fitCurve = (): void => {
+        const firstPoint = this._points[0];
+        this._path2D = new Path2D();
+        this._isPath2DValid = true;
+        this._path2D.moveTo(firstPoint.getX(), firstPoint.getY());
+
+        this._fittedCurve = CurveFitter.Fit(this._points);
+        this._path2D.quadraticCurveTo(this._fittedCurve.control.getX(), this._fittedCurve.control.getY(), this._fittedCurve.end.getX(), this._fittedCurve.end.getY());        
+    }
```

Comment:
missing semis at the end of method definitions in this file

Reply:
Thanks!